go.mod

```diff
@@ -4,21 +4,31 @@ go 1.20
 
 require (
 	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.82
-	github.com/edgexfoundry/go-mod-core-contracts/v3 v3.0.0-dev.39
+	github.com/edgexfoundry/go-mod-core-contracts/v3 v3.0.0-dev.40
 	github.com/stretchr/testify v1.8.2
 	github.com/vladimirvivien/go4vl v0.0.5
 	github.com/xfrr/goffmpeg v0.0.0-20210624103149-5ca2d3062daf
 )
 
+require (
+	github.com/edgexfoundry/go-mod-configuration v0.0.8 // indirect
+	github.com/edgexfoundry/go-mod-core-contracts v0.1.147 // indirect
+	github.com/edgexfoundry/go-mod-secrets v0.0.32 // indirect
+	github.com/go-kit/kit v0.9.0 // indirect
+	github.com/pelletier/go-toml v1.2.0 // indirect
+)
+
 require (
 	github.com/Microsoft/go-winio v0.6.0 // indirect
 	github.com/OneOfOne/xxhash v1.2.8 // indirect
 	github.com/armon/go-metrics v0.3.10 // indirect
 	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/eclipse/paho.mqtt.golang v1.4.2 // indirect
-	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.80 // indirect
+	github.com/edgexfoundry/go-mod-bootstrap v0.0.72
+	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.81 // indirect
```

Comment:
there is something off about this go.mod file. why is it using both v0 and v3 go-mod-bootstrap? Also, there should only be 2 sections and not 3.

Reply:
> there is something off about this go.mod file. why is it using both v0 and v3 go-mod-bootstrap? Also, there should only be 2 sections and not 3.

Wow, that is interesting. @brianointel you need to clear the v0.0.72 from your module cache and find the import that is referencing `github.com/edgexfoundry/go-mod-bootstrap` and fix it to be `github.com/edgexfoundry/go-mod-bootstrap/v3`